Blizzard.IO.RabbitMQ.Builders/NetqRabbitPublisherBuilder.cs

```diff
@@ -0,0 +1,110 @@
+﻿using Blizzard.IO.Core;
+using Blizzard.IO.RabbitMQ.Entities;
+using Blizzard.IO.RabbitMQ.Extensions;
+using Blizzard.IO.Serialization.Json;
+using EasyNetQ;
+using Microsoft.Extensions.Logging;
+
+namespace Blizzard.IO.RabbitMQ.Builders
+{
+    public class NetqRabbitPublisherBuilder<TData> : BaseNetqRabbitBuilder
+    {
+        private RabbitExchange _destinationExchange = null;
+        private ISerializer<TData> _serializer = new JsonSerializer<TData>();
+        private string _routingKey = "/";
+        private bool _isAbstract;
```

Comment:
Initialize this field

Reply:
The default of bool is false, so there is no need to initialize it

Reply:
I know but make it more readable

Reply:
f